app/views/service_requests/right_navigation/_short_interaction_modal.html.haml

```diff
@@ -0,0 +1,64 @@
+-# Copyright © 2011 MUSC Foundation for Research Development
+-# All rights reserved.
+
+-# Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+
+-# 1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+
+-# 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following
+-# disclaimer in the documentation and/or other materials provided with the distribution.
+
+-# 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products
+-# derived from this software without specific prior written permission.
+
+-# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING,
+-# BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+-# SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+-# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+-# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+-# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#interaction-modal
+  .modal-dialog.interaction
+    .modal-content
+      = form_for @short_interaction, url: short_interactions_path, method: :post, remote: true do |f|
+        = f.hidden_field :identity_id, value: @identity.id
+        .modal-header
+          %button.close{ type: 'button', data: { dismiss: 'modal' } }
+            %span{ aria: { hidden: 'true' } } &times;
+          %h4.text-center#modal-title
+            = t(:proper)[:right_navigation][:short_interaction][:header]
+        .modal-body
+          #modal_errors
+          .form-group.row
+            = f.label :subject, t(:proper)[:right_navigation][:short_interaction][:subject], class: 'col-lg-4 control-label'
+            .col-lg-8
+              = f.select :subject, options_for_select(PermissibleValue.get_inverted_hash('interaction_subject')), :prompt =>t(:proper)[:right_navigation][:short_interaction][:subject_select], class: 'form-control'
```

Comment:
From manual testing, it looks like this select isn't correctly applying the Bootstrap `form-control` class. Adding curly brackets around the `prompt` attribute fix the problem.